lib/DocblockParser/Ast/Tag/SeeTag.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Phpactor\DocblockParser\Ast\Tag;
+
+use Phpactor\DocblockParser\Ast\TagNode;
+use Phpactor\DocblockParser\Ast\TextNode;
+use Phpactor\DocblockParser\Ast\Token;
+
+class SeeTag extends TagNode
+{
+    public const CHILD_NAMES = [
+        'token',
+        'text',
```

Comment:
Note that [`@see`](https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc-tags.md#513-see) has an FQSEN/URL and an optional description

Reply:
Yes, that is correct. But as far as I know we don't have a way to reference members here. So it might as well will be text.